Log redact on TiFlash and PD (4.0) (#4997)
pingcap/docs: pd-configuration-file.md

+ The path of the PEM file that contains the X509 key
+ Default value: ""

### `redact-info-log` <span class="version-mark">New in v4.0.10</span>

+ Enables or disables log redaction in PD.

> + If the configuration value is set to `true`, all user data in the PD log will be redacted.

Reviewer: Do you think we should align this with v5.0? https://docs.pingcap.com/tidb/v5.0/pd-configuration-file#redact-info-log-new-in-v500-rc
The description is not consistent.

Author: This is made inconsistent on purpose. This `redact-info-log` is introduced in v5.0.0-rc for dev/v5.0 and in v4.0.10 for v4.0. v5.0.0-rc is earlier than v4.0.10.

> + Enables or disables log redaction in PD.
> + If the configuration value is set to `true`, all user data in the PD log will be redacted.

Author: ```suggestion
+ Controls whether to enable log redaction in the PD log.
+ When you set the configuration value to `true`, user data is redacted in the PD log.
```
